source/features/forked-to.tsx

```diff
@@ -0,0 +1,154 @@
+import './more-dropdown.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL, getUsername} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const currentRepo = getRepoURL();
+
+async function init(): Promise<void> {
```

Comment:
Can you move init before features.add? All features are like that. I tend to follow this rule even if it’s not enforced automatically

https://eslint.org/docs/2.0.0/rules/no-use-before-define